### PR TITLE
Manually set names and prettier Json configs

### PR DESCRIPTION
**Pull request by rachitnigam**

### CODE REVIEW

File: polyphemus/db.py [hunk 1]
```diff
@@ -177,6 +177,15 @@ def set_state(self, job, state):
...
             self.cv.notify_all()
 
+    def set_name(self, job, name):
+        """Update a job's name
+        """
+        with self.cv:
+            job['config']['hwname'] = name
+            self.log(job['name'], 'hwname changed to {}'.format(name))
+            self._write(job)
+            self.cv.notify_all()
```
[the remark this answers is not in the document]
Comment (rachitnigam) [reply polyphemus/db.py]: Got it. This was just copy pasting from the code above.

File: polyphemus/db.py [hunk 2]
```diff
@@ -177,6 +177,15 @@ def set_state(self, job, state):
             self._write(job)
             self.cv.notify_all()
 
+    def set_name(self, job, name):
```
[the remark this answers is not in the document]
Comment (rachitnigam) [reply polyphemus/db.py]: Fixed. Is it fine to use the `db._write` function?

[the remark this answers is not in the document]
Comment (sampsyo) [reply polyphemus/db.py]: Yep, seems fine! Maybe I should make that public after all.

[the remark this answers is not in the document]
Comment (rachitnigam) [reply polyphemus/db.py]: Will fix that in a different PR.

**Pull request merged**